docs/apps/crypto-trading.md

```diff
@@ -39,153 +51,163 @@ whereas the strategy is kept in trades collection.
 /**
 Testing the Stream Application:
     1. Publish the app
+       
     2. Start the GUI against the same federation
```

Comment:
Not sure what this procedure here is for, didn't you give steps for this already? Either way, rewrite to remove "federation."